R/unsupervised_segment.R

```diff
@@ -323,9 +323,9 @@ lambdaestimation <- function(x, num.method = 1, fac.method = 1, outtype = "numer
             Rather use kmeans() instead of kprotos().")
     anyfact <- FALSE
   } 
-  if(num.method == 1 & verbose == TRUE) cat("Numeric variances:\n")
-  if(num.method == 2 & verbose == TRUE) cat("Numeric standard deviations:\n")
-  print(vnum)
+  # if(num.method == 1 & verbose == TRUE) cat("Numeric variances:\n")
+  # if(num.method == 2 & verbose == TRUE) cat("Numeric standard deviations:\n")
+  # print(vnum)
```

Comment:
Why do these now need to be commented out? Because it will try and generate them for customerid?

Reply:
I commented out only to avoid printing too much info. Now it only prints Average numeric variance, Average numeric standard deviation, Heuristic for categorical variables, Average categorical variation and Estimated lambda.